pandas/core/base.py

```diff
@@ -990,7 +990,7 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
really? that is odd, ok then pls add to the doc-strings where you added it with a comment that this is for compatiblity only.

Further that would be a bug in numpy. It shouldn't be passing positional arguments to sub-class likes.